backend/functions/src/email/templates.ts

```diff
@@ -0,0 +1,257 @@
+  </div>
+
+    <br>
+    <br>
+
+    Happy studying!`
+}
+
+const firstNoMatch = (className: string) => {
+  return `< div > Dear Student, </>
+    < br >
+    <br>
+
+    <div>
+    We are writing to let you know that we are still working on finding study partners
+  for the class(es) you requested: <b>${className} </b>. We'll be in touch soon with an update!
+    In the meantime, the < a href = "http://lsc.cornell.edu/" > LSC website < /a> offers lots of
+    information about general study skills, learning effectively, and tutoring.
+  < /div>`
+}
+
+const secondNoMatch = () => {
+  return `<div> Dear Student, </div>
+    <br> 
+    <br> 
+
+  <div> 
+    We are writing to follow up and let you know that unfortunately we have not 
+    been successful, so far, in finding study partners for the class(es) you 
+    requested. We’ll be in touch again if another student requests study partners
+    for your class(es). In the meantime, the LSC website offers lots of information
+    about general study skills, learning effectively, and tutoring. 
+  </div>`
+}
+
+const addStudent = () => {
+  return `<div> Dear Students,  </div>  
+    <br> 
+    <br> 
+  
+  <div> 
+    We are writing to check in about your study partners. Although our hope is to
+    form groups with the same composition for the entire semester, that’s not 
+    always possible. 
+    Another student has requested study partners for your class, and we have 
+    not been able to find a match for them. We’d like to ask that you please 
+    consider adding this fellow student to your group. Please let us know if you 
+    have any questions or concerns. If we don’t hear back from you we’ll connect
+    you all via email. Thank you! 
+  </div>
+ `
+}
+
+const lateAddStudent = () => {
+  return ` 
+  <div> Dear Students, </div> 
+    <br> 
+    <br> 
+
+  <div> 
+
+    We are writing to check in about your study partners. Although our hope is to
+    form groups with the same composition for the entire semester, that’s not 
+    always possible. 
+        <br> 
+        <br> 
+    Another student has requested study partners for your class, and we have 
+    not been able to find a match for them. Although it’s late in the semester, we
+    are hoping that you would please consider adding this fellow student to your 
+    group. Please let us know if you have any questions or concerns. We know 
+    that it can be great to get new perspectives when you’re studying, and we 
+    also understand if at this point in the semester your group feels that adding 
+    another student would not be beneficial. If we don’t hear back from you in 
+    the next 3 business days, we’ll connect you all via email. Thank you!
+
+  </div>`
+}
+
+const askJoinGroup = (newStudent: string, studentsRaw: string[]) => {
+  const parseNames = () => {
+    let res = ''
+    studentsRaw.forEach((e: string) => {
+      res += ', ' + e
+    })
+
+    return res
+  }
+
+  const students = parseNames()
```

Comment:
Is this the same code as `parseRecipients`? See the comment there, also this results in `, a, b`